build-scripts/swaync.rpkg.spec

```diff
@@ -84,9 +84,9 @@ This package installs Fish completion files for %{name}
 %{_bindir}/swaync-client
 %{_bindir}/swaync
 %license COPYING
-%config %{_sysconfdir}/xdg/swaync/configSchema.json
-%config %{_sysconfdir}/xdg/swaync/config.json
-%config %{_sysconfdir}/xdg/swaync/style.css
+%config(noreplace) %{_sysconfdir}/xdg/swaync/configSchema.json
+%config(noreplace) %{_sysconfdir}/xdg/swaync/config.json
+%config(noreplace) %{_sysconfdir}/xdg/swaync/style.css
```

Comment:
Editing these files are generally discouraged, specially when swaync can parse configs in `XDG_CONFIG_HOME`